### PR TITLE
feat: enclosure image and episode list

### LINKED ISSUE #42: [APP] - Add item list
Add new tab with episodes, when a click is done over the episode a form with the following fields is displayed.
The form also contains the update logic.

- [x] title
- [x] link (href)
- [x] description
- [x] guid (#text + isPermaLink)
- [ ] pubDate
- [x] enclosure
- [x] itunes:author
- [x] itunes:subtitle
- [x] itunes:summary
- [x] itune:duration
- [x] itunes:explicit
- [x] itunes:image (href)
- [x] itunes:season
- [x] itunes:episode (should be prepopulated by previous episode + 1)
- [ ] itunes:episodeType

Verify of those fields are available in the core/schema and if missing add it

### PR DESCRIPTION
**Pull request by brainrepo**
closes #42

### CODE REVIEW

File: packages/app/src-tauri/src/commands/ftp.rs [hunk 1]
```diff
@@ -19,14 +20,21 @@ pub struct Payload {
     https: bool,
 }
 
+#[derive(serde::Serialize)]
```
Comment (brainrepo) [inline packages/app/src-tauri/src/commands/ftp.rs:23]: @newfla can you review it pls?

File: packages/app/src-tauri/src/commands/s3.rs [hunk 2]
```diff
@@ -77,10 +86,11 @@ async fn s3_upload_internal(payload: Payload, use_path_style: bool) -> Result<St
         format!("{}.{}", payload.file_name, ext)
     };
 
-    Ok(format!(
-        "{}://{}/{}",
-        protocol, payload.http_host, file_path
-    ))
+    Ok(UploadResult {
```
Comment (brainrepo) [inline packages/app/src-tauri/src/commands/s3.rs:89]: Same as above

File: packages/app/src-tauri/src/commands/fs.rs [hunk 3]
```diff
@@ -14,11 +16,55 @@ async fn read_text_file_internal(path: &str) -> Result<String> {
...
 
+#[derive(serde::Serialize)]
+pub struct FileInfo {
+    content_type: String,
+    content_length: String,
+}
+
+#[named]
+#[tauri::command]
+pub async fn read_file_info(url: &str) -> Result<FileInfo> {
```
[the remark this answers is not in the document]
Comment (brainrepo) [reply packages/app/src-tauri/src/commands/fs.rs:26]: Moved to the file module

**Comment by brainrepo** [comment]
@newfla thank you for your detailed review, I applied your suggestions. 
I moved the file_info logic into a `file` module.

**Review by newfla — APPROVED** [review]
LGTM! Amazing job @brainrepo

**Review by newfla — APPROVED** [review]

**Pull request merged**